server/controllers/post.controller.js

```diff
@@ -0,0 +1,81 @@
+const { where, Op } = require('sequelize')
+const db = require('../models')
+const Post = db.posts
+
+exports.deleteOnePost = async (req, res) => {
+    await Post.destroy({ where: [{ user_id: res.locals.userID }, { id: req.body.id }] })

```

Comment:
`req.body.id` is a unique identifier of a blog, so don't need to include user_id in where clause